backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -438,6 +438,33 @@
           end
         end
       end
+
+      describe 'line item sort order' do
+        let(:product2) { create(:product) }
+        let(:product3) { create(:product) }
+
+        before do
+          # grab this one first and then create others that should end up before
+          # and after
+          @middle_line_item = order.line_items[0]
+
+          @first_line_item  = order.contents.add(product2.master)
+          @first_line_item.update_columns(created_at: 1.day.ago)
+          @last_line_item  = order.contents.add(product3.master)
+          @last_line_item.update_columns(created_at: 1.day.from_now)
+        end
+
+        it 'orders the items in a shipment by created_at' do
+          visit spree.edit_admin_order_path(order)
+
+          stock_items = page.all(:css, '.stock-item')
```

Comment:
`all` isn't safe to use in feature specs, since it's possible for it to return `[]` if the desired items aren't on the page (ex. not yet fully loaded).

`page.all(:css, '.stock-item', count: 3)` should work